src/Core/Analysis/CleanCodeTaxonomy.cs

```diff
@@ -98,8 +101,8 @@ public enum SoftwareQuality
     /// </summary>
     public enum SoftwareQualitySeverity
     {
-        High,
+        Low,
```

Comment:
This ordering is rather weak. Either add explicit values for enum or add a custom IComparer